docs/src/main/paradox/scala/http/client-side/request-level.md

```diff
@@ -66,7 +73,7 @@ It creates a new "super connection pool flow", which routes incoming requests to
 depending on their respective effective URIs.
 
 The `Flow` returned by `Http().superPool(...)` is very similar to the one from the @ref[Host-Level Client-Side API](host-level.md), so the
-@ref[Using a Host Connection Pool](host-level.md#using-a-host-connection-pool) section also applies here.
+@scala[@ref[Using a Host Connection Pool](host-level.md#using-a-host-connection-pool)]@java[@ref[Using a Host Connection Pool](host-level.md#using-a-host-connection-pool-java)] section also applies here.
```

Comment:
Another PR unifies this page. Maybe I can just drop the suffix "java" here and the other PR will do the unification